examples/minikube/http-sw-app.yaml

```diff
@@ -31,42 +31,26 @@ spec:
       - name: deathstar
         image: cilium/starwars
 ---
-apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: v1
+kind: Pod
 metadata:
   name: tiefighter
+  labels:
+    org: empire
+    class: tiefighter
 spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      org: empire
-      class: tiefighter
-  template:
-    metadata:
-      labels:
-        org: empire
-        class: tiefighter
-    spec:
-      containers:
-      - name: spaceship
-        image: tgraf/netperf
+  containers:
+  - name: spaceship
+    image: tgraf/netperf
 ---
-apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: v1
```

Comment:
Can you please verify that these YAML changes work on all documented versions which Cilium supports?

Reply:
Sorry, yes, K8s.

Reply:
Tested using minikube 0.26.1 for following k8s versions. I have only tested that the yaml creates the app successfully, all pods running and the http calls between them working as expected. I have tested the full example only on 1.10
1.7.3
1.8.0
1.9.0
1.10.0